dfget/core/downloader/p2p_downloader/power_client.go

```diff
@@ -111,6 +111,7 @@ func (pc *PowerClient) downloadPiece() (content *bytes.Buffer, e error) {
 		}
 	}
 
+	logrus.Infof("start to download piece from %s:%d with taskID(%s)", dstIP, peerPort, pc.taskID)
```

Comment:
Why does add this info log? I remember that there is already an info log to output the similar information. Could you confirm again?

Reply:
I have not found the similar log anywhere else. 😄

Reply:
There are already more detailed debug logs to output the information. I think you can just choose one and change its level to info if really necessary.

Reply:
Yeah. I got it. https://github.com/dragonflyoss/Dragonfly/blob/master/dfget/core/downloader/p2p_downloader/power_client.go#L77 

And then I will delete this log.

Reply:
Done. PTAL again. THX>